packages/driver/cypress/e2e/dom/visibility.cy.ts

```diff
@@ -53,7 +53,7 @@ describe('src/cypress/dom/visibility', () => {
       expect(fn()).to.be.true
     })
 
-    it('returns false window and body > window height', () => {
+    it('returns false if window and body < window height', () => {
```

Comment:
what this title wrong before?

Reply:
Yes. Things should only be scrollable if they're > than the height not <.

Reply:
Basically this is the opposite of line 45.